packages/@uppy/dashboard/src/index.js

```diff
@@ -546,7 +546,17 @@ module.exports = class Dashboard extends Plugin {
     })
 
     // 4. Add all dropped files
-    getDroppedFiles(event.dataTransfer)
+    let executedDropErrorOnce = false
+    const logDropError = (error) => {
+      this.uppy.log(error.message, 'error')
```

Comment:
Is there a reason for logging `error.message` specifically, maybe log the whole error object?

Reply:
Good call, changing